apps/wallet/src/ui/app/components/accounts/ProtectAccountForm.tsx

```diff
@@ -58,7 +66,16 @@ export function ProtectAccountForm({
 			/>
 			<div className="flex flex-col gap-4">
 				<CheckboxField name="enabledAutolock" label="Auto-lock after I am inactive for" />
-				{/* TODO: Abhi is working on designs for the auto-lock input, we'll add this when it's ready */}
+				<div className="flex items-center justify-between gap-2">
+					<NumberField type="number" {...register('autoLockTimer')} />
+					<SelectDropdown
+						options={lockIntervals}
+						placeholder={'Hour'}
```

Comment:
is this a placeholder or default value?

```suggestion
						placeholder="Hour"
```